frontend/app/services/MemberService.scala

```diff
@@ -539,7 +535,7 @@ class MemberService(identityService: IdentityService,
         Account.payPal(contactId, currency, autopay = true)
     }
 
-  private def createPaymentMethod(contactId: ContactId, email: Option[String], paymentForm: CommonPaymentForm, stripeCustomer: Option[Customer]) =
```

Comment:
Note that `contactId: ContactId` was not really a necessary parameter on this method :smile: